src/app/Main/Header/Export/Header-Export.component.tsx

```diff
@@ -13,21 +13,21 @@ export function HeaderExportComponent(props: {
     settings: ExportInterface;
     buttonLabel: JSX.Element;
     exportLabels: ExportLabels;
-    handleClickOpen: () => void;
-    handleClickClose: () => void;
+    onClickOpen: () => void;
+    onClickClose: () => void;
     handleCheckbox: (name: string, checked: boolean) => void;
 }): JSX.Element {
     const { filter } = useContext(FilterContext);
 
     return (
         <div>
             <HeaderExportButtonComponent
-                handleClickOpen={props.handleClickOpen}
+                onClickOpen={props.onClickOpen}
```

Comment:
you could use handleClick function in this file to. So you have the same convention in all files